homework24/script.js

```diff
@@ -17,44 +17,44 @@ class Hamburger {
     };
 
     addTopping(topping) {
-        if(topping === 'sauce') {
+        if(topping === Hamburger.TOPPING_SAUCE) {
             this._price += 15;
-        } else if (topping === 'mayo') {
+        } else if (topping === Hamburger.TOPPING_MAYO) {
             this._price += 20;
             this._calories += 5;
         }
 
     };
 
     calculate() {
-        if (this._size === 'small') {
+        if (this._size ===  Hamburger.SIZE_SMALL) {
```

Comment:
Let's refactor a bit

Move all conditions to separate method

```
getCalories(type) {
  switch (type) {
    case Hamburger.SIZE_SMALL:
      return 20;
    case Hamburger.SIZE_LARGE:
      return 40;
      // ... etc
      default:
        return 0;
  }
}
```

then in `calculate` use it

```
this._calories += this.getCalories(this._size);
this._calories += this.getCalories(this._stuffing);
```